### ISSUE TITLE
shutdown_all_and_replace_node_test failed: Missing: ['Starting listening for CQL clients']

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylla#4346]
scylla version c7d05b88a6562ea7b167a0e8e7d02cef9dfa6dd6
scylla-dtest version scylladb/scylla@a7482541884da61c76c5329d81f4efdad95b7799

See [dtest-release/62/testReport/replace_address_test/TestReplaceAddress/shutdown_all_and_replace_node_test](http://jenkins.cloudius-systems.com:8080/view/master/job/scylla-master/job/dtest-release/62/testReport/replace_address_test/TestReplaceAddress/shutdown_all_and_replace_node_test/):
```
Error Message
19 Mar 2019 00:47:54 [node4] Missing: ['Starting listening for CQL clients']:
```

From the node [logs](http://jenkins.cloudius-systems.com:8080/view/master/job/scylla-master/job/dtest-release/62/artifact/logs-release.2/1552956474526_replace_address_test.TestReplaceAddress.shutdown_all_and_replace_node_test/) it looks like nodes 1 and 2 did not shutdown properly.

**Comment by asias** [comment]
If it is shutdown issue, it might be: https://github.com/scylladb/scylla/issues/4044

FYI: The scylla fix is not merged yet: [PATCH scylla v1] storage_service: Allow replacing node when all nodes were shutdow

**Comment by denesb** [comment]
It doesn't look like a shutdown issue, the node apparently doesn't even start to the point of opening the CQL port.

**Comment by asias** [comment]
> It doesn't look like a shutdown issue, the node apparently doesn't even start to the point of opening the CQL port.

You mean n4? It is expected since the fix is not merged yet.

**Comment by denesb** [comment]
The node which won't start logs this:
```
INFO  2019-03-19 02:38:21,136 [shard 0] gossip - FatClient 127.0.48.3 has been silent for 30000ms, removing from gossip
INFO  2019-03-19 02:38:21,136 [shard 0] gossip - removed 127.0.48.3 from _seeds, updated _seeds list = {127.0.48.1, 127.0.48.2}
ERROR 2019-03-19 02:38:51,051 [shard 0] seastar - Exiting on unhandled exception: std::runtime_error (Cannot replace token -3563940604998535138 which does not exist!)
```
@asias you have a patch on the ML [1] which fixes this, no?

[1] "[PATCH scylla v1] storage_service: Allow replacing node when all nodes were shutdown" https://groups.google.com/d/msg/scylladb-dev/YJqyiR_vYno/Qflp1WTUCQAJ

**Comment by denesb** [comment]
Yes, n4.

**Comment by denesb** [comment]
Oh I see now that you also mentioned this patch. I don't know how I didn't see that. :man_facepalming:

**Comment by asias** [comment]
OK, I guess we are in the same page now ;-)

**Comment by slivne** [comment]
maybe related #4361

**Comment by asias** [comment]
This is not a regression and we do not have agreement on the fix, so maybe 3.2?

**Comment by asias** [comment]
Closing. It is duplication of https://github.com/scylladb/scylla/issues/4325.
